integration/proxy/teleterm_test.go

```diff
@@ -232,3 +233,129 @@ func (c *mockTSHDEventsService) SendNotification(context.Context, *api.SendNotif
 	c.callCounts["SendNotification"]++
 	return &api.SendNotificationResponse{}, nil
 }
+
+// TestTeletermKubeGateway tests making kube API calls against Teleterm kube
+// gateway and reissuing certs.
+//
+// Note that this test does NOT reuse existing kube test setups as IP Pinning
+// is enabled in those tests. Use certs with pinned IPs are injected during
```

Comment:
I think it was supposed to be "user certs" not "use certs", correct?

```suggestion
// is enabled in those tests. User certs with pinned IPs are injected during
```